app/src/main/res/values/strings.xml

```diff
@@ -290,6 +290,20 @@
     <string name="admin_reason">Reason: %s</string>
     <string name="admin_reportedBy">Reported By: %s</string>
     <string name="admin_description">Description: %s</string>
+    <string name="more_options">More Options</string>
+    <string name="options_for_report">Details for this Reported Object:</string>
+    <string name="choose_action_for_report">Choose an action for this report</string>
+    <string name="Return">Return</string>
```

Comment:
That is just a minor detail but try to rename it `return` to keep consistency with the other strings.

Reply:
I can't, since return is a reserved word in Java. I would have liked to!

Reply:
Maybe you can try `re_turn` xD. It is a joke I understand you can let it that way we can't really do anything about it.